src/client/scripts/cardSubmitArea.js

```diff
@@ -5,8 +5,12 @@ Template.cardSubmitArea.events({
 
     "submit #card": function(eve){
         eve.preventDefault();
+        var author;
 
-        var author = eve.target.author.value;
+        if(!Meteor.user())
```

Comment:
I never used a ternary before so not quite sure what they are or how to use them but ill look into that